mne/preprocessing/ica.py

```diff
@@ -811,7 +822,7 @@ def _export_info(self, info, container, add_channels):
         ch_names = []
         ch_info = info['chs'] = []
         for ii in range(self.n_components_):
-            this_source = 'ICA %03d' % (ii + 1)
+            this_source = 'ICA %03d' % (ii)
```

Comment:
we could really just make clear that these are the indices. We have a similari issue in #4049 and should solve it consistently.